autobotAI_integrations/integrations/openai/ai_evaluator_code.py

```
@@ -1,15 +1,62 @@
+        
+        # Ensure we have at least 1 resource, even if it exceeds limit slightly
+        if parsable_resources_count == 0 and len(resources) > 0:
+            logger.warning(f"All resources exceed token limit. Using first resource anyway.")
+            parsable_resources_count = 1
+        
+        logger.info(f"Using {len(resources)} resources. Token count: {current_token_count}/{SAFE_INPUT_TOKENS}")
+        
+        if len(resources) == 0:
+            logger.error("No resources available after filtering. Original resources: {len(resources)}")
+        
+    except Exception as e:
```

Ruff (0.15.15)

[warning] Do not catch blind exception: `Exception`

(BLE001)